src/style.css

```diff
@@ -0,0 +1,90 @@
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  min-width: 300px;
+  flex-basis: 300px;
+  flex-grow: 1;
+  font-size: 100px;
+  color: #fff;
+}
+
+.block:first-child {
```

Comment:
Interesting solution with nth-child but let's rewrite this one. Add class block-1, block-2 ...